src/components/animations/AnimatedSpinner.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Animated, { Easing, useAnimatedReaction, useAnimatedStyle, useSharedValue, withRepeat, withTiming } from 'react-native-reanimated';
+import { useForegroundColor } from '@/design-system';
+import styled from '@/styled-thing';
+import { ImgixImage } from '@/components/images';
+import Spinner from '../../assets/chartSpinner.png';
```

Comment:
should this be swap spinner?

Reply:
meant for this component to be used elsewhere and chartSpinner.png is better suited to most cases, so left that in as the default and passed in a custom src for the swap spinner